frontend-ssr/redux/exercise-submissions/exercise-submissions.reducer.ts

```diff
@@ -53,6 +54,7 @@ export const submissionReducer = (state = initialState, action: { type: string;
           ? [...state.submissions, ...newSubmissions]
           : newSubmissions,
         page: state.page + 1,
+        end: true,
```

Comment:
Why are we hardcoding `end: true` here? This means that after the first fetched page we're done.

But what if there are more pages?